README.md

```diff
@@ -63,1498 +104,468 @@ Delegating a capability is like the statement `message`. Task is akin to `messag
-│                                                                  │         │    ┆         │         │   │
-│                                                                  │   Dan   ╞════Set DNS══►│  Erin   │   │
-│                                                                  │         │              │         │   │
-│                                                                  └─────────┘              └─────────┘   │
-│                                                                                                         │
-└─────────────────────────────────────────────────────────────────────────────────────────────────────────┘
-```
-
-## 1.3 A Note On Serialization
-
-The JSON examples below are given in [DAG-JSON], but UCAN Task is actually defined as IPLD. This makes UCAN Task agnostic to encoding. DAG-JSON follows particular conventions around wrapping CIDs and binary data in tags like so:
-
-### CID
-
-```json
-{"/": "Qmf412jQZiuVUtdgnB36FXFX7xg5V6KEbSJ4dpQuhkLyfD"}
-```
-
-### Bytes
-
-```json
-{"/": {"bytes": "s0m3Byte5"}}
-```
+## Public Resources
+[Public Resources]: #public-resources
 
-This format help disambiguate type information in generic [DAG-JSON] tooling. However, your presentation need not be in this specific format, as long as it can be converted to and from this cleanly. As it is used for the signature format, [DAG-CBOR] is RECOMMENDED.
+A core part of UCAN's design is interacting with the wider, non-UCAN world. Many resources are open to anyone to access, such as unauthenticated web endpoints. Unlike UCAN-controlled resources, an invocation on public resources is both possible, and a hard requirement for initiating a flow (e.g. sign up). These cases typically involve a reference passed out of band (such as a web link). Due to [designation without authorization], knowing the URI of a public resource is often sufficient for interacting with it. In these cases, the Executor MAY accept Invocations without having a "closed-loop" proof chain, but this SHOULD NOT be the default behavior.
 
-## 1.4 Signatures
+## Promise Pipelining
+[Promise Pipelining]: #promise-pipelining
 
-All payloads described in this spec MUST be signed with a [Varsig].
+[UCAN Promise] extends UCAN Invocation with [distributed promise pipelines]. Promises are helpful in a wide variety of situations for efficiency and convenience. Implementations supporting UCAN Promises is RECOMMENDED.
 
-# 2 High-Level Concepts
+# Concepts
+[Concepts]: #concepts
 
-## 2.1 Roles
+## Roles
+[Roles]: #roles
 
 Task adds two new roles to UCAN: invoker and executor. The existing UCAN delegator and delegate principals MUST persist to the invocation.
 
-| UCAN Field | Delegation                             | Task                            |
-| ---------- | -------------------------------------- | ------------------------------- |
+| UCAN Field | Delegation                             | Invocation                      |
```

Comment:
> The existing UCAN delegator and delegate principals MUST persist to the invocation.

I'd love this to be unpacked a bit, it's too terse for me.  Are you saying that the _same identifiers_ used from principles at delegation time must be _dereferenceable_ at invocation time? I feel like phrasing that in terms of local context or dereferencing is prefereable to making it sound more existential (and untestable) as some kind of... survival in a hostile [distributed] environment 😅